From a8c4a92ff33bd2dff205b9c07931992da303dc47 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 29 Mar 2018 23:58:02 +0000
Subject: [PATCH] Install iscsistart

dracut requires iscsistart to be packaged so add iscsistart to the
set of PROGRAMS files.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c8cd00e..42a78f2 100644
--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,7 @@
 MANPAGES = doc/iscsid.8 doc/iscsiadm.8 doc/iscsi_discovery.8 \
 		iscsiuio/docs/iscsiuio.8 doc/iscsi_fw_login.8 doc/iscsi-iname.8 \
 		doc/iscsistart.8
-PROGRAMS = usr/iscsid usr/iscsiadm utils/iscsi-iname iscsiuio/src/unix/iscsiuio
+PROGRAMS = usr/iscsid usr/iscsiadm utils/iscsi-iname iscsiuio/src/unix/iscsiuio usr/iscsistart
 SCRIPTS = utils/iscsi_discovery utils/iscsi_fw_login utils/iscsi_offload \
 		  utils/iscsi-gen-initiatorname
 INSTALL = install
